emrun.py

```diff
@@ -1622,6 +1672,10 @@ def run():
   else:
     url = os.path.relpath(os.path.abspath(file_to_serve), serve_dir)
 
+  # If there are any --prepend-js-in-html arguments, convert the paths to absolute paths before changing the directory,
+  # so that the --prepend-js-in-html arguments can be located outside the web server root.
+  emrun_options.prepend_js_in_html = [os.path.join(os.getcwd(), f[0]) for f in emrun_options.prepend_js_in_html]
```

Comment:
`[os.path.abspath(f) for f in emrun_options.prepend_js_in_html]`

(I can't see why f[0] makes sense here in existing code.. isn't f[0] the first char of each path?)